### PR TITLE
Add aws-cli to infrastructure stuff into Brewfile

### PR DESCRIPTION
**Pull request by serena-piccioni**

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -48,6 +49,8 @@ tap "heroku/brew"
 brew "tfenv"
 brew "tflint"
 brew "heroku/brew/heroku"
+brew "awscli"
+brew "kubernetes-cli"
```
Comment (erbridge) [inline Brewfile:53]: You run these yourself? Do we use kubernetes anywhere?

File: Brewfile [hunk 2]
```diff
@@ -37,6 +37,7 @@ brew "dxw/tap/whippet"
 brew "shellcheck"
 
 # JS stuff
+brew "grunt-cli"
```
Comment (erbridge) [inline Brewfile:40]: Strong disagree on this one. You should be using `npx grunt` instead or you'll end up with mismatched versions.

**Review by erbridge — CHANGES_REQUESTED** [review]

File: Brewfile (hunk 1 above)
Comment (erbridge) [reply Brewfile:53]: I can get behind `awscli` but probably not Kubernetes unless we're actually using it at all.

Comment (serena-piccioni) [reply Brewfile:53]: Err sorry, we don't use Kubernetes, I definitely need another coffee.

**Review by erbridge — APPROVED** [review]

**Pull request merged**